data/jainankit0811.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Ankit Dhanawat",
+  "type": "personal",
+  "bio": "Student"
+},
```

Comment:
Hi @jainankit0811,
The JSON is invalid. On line 5 you don't need to use the `}` bracket

You can test the JSON here on Linkfree Playground.
-> https://linkfree.io/playground

Here is a tool for JSON Lint which I use,
-> https://jsonlint.com/

Thanks :)